packages/viewer/src/lib/interactions/interactionApi.ts

```diff
@@ -310,6 +310,16 @@ export class InteractionApi {
     });
   }
 
+  /**
+   * Performs a view all operation for the scene's bounding box, and requests a new image
+   * for the updated scene.
+   */
+  public async viewAll(): Promise<void> {
+    return this.transformCamera(({ camera, viewport }) => {
+      return camera.viewAll();
+    });
```

Comment:
You can simplify this to `return this.getScene().camera().viewAll().render();`. Going through a camera transform will put issue additional renders for intermediate frames, which probably isn't required here.